src/sdk.js

```diff
@@ -60,6 +59,22 @@ class OpenForm {
     this.formObject = null;
     this.lang = lang;
 
+    switch (typeof languageSelectorTarget) {
+      case "string": {
+        this.languageSelectorTarget = document.querySelector(
+          languageSelectorTarget
+        );
+        break;
+      }
+      case "object": {
+        this.languageSelectorTarget = languageSelectorTarget;
+        break;
+      }
+      default:
+        this.languageSelectorTarget = undefined;
+        break;
+    }
+
```

Comment:
Where should tests for this live? I've tested it manually by toying around with `public/sdk.html` but that didn't feel quite right.

Reply:
`sdk.spec.js` would be appropriate and be picked up - you can find some inspiration in `jstests/formio/components/`